dvtemplates/Dockerfile

```diff
@@ -0,0 +1,3 @@
+#This docker file is used in openshift CI
+FROM scratch
+COPY Fedora*qcow2 /disk/
```

Comment:
```suggestion
# Base for the fetching of the qcow is this image which comes from the openshift registry
FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13 AS builder
RUN curl -O /tmp/Fedora.qcow2 http://your/url/goes/here

FROM scratch
COPY --from=builder /tmp/Fedora.qcow2 /disk/
```

Reply:
Using Internal Prow for now, not openshift CI